Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM ubuntu:16.04
+FROM golang:latest as builder
 
-COPY bin/gargantua /usr/local/bin
+WORKDIR /go/gargantua/
+COPY . .
+ENV GOOS=linux 
+ENV CGO_ENABLED=0 
+RUN go build -o bin/gargantua .
+RUN touch /tmp
 
-ENTRYPOINT /usr/local/bin/gargantua -v=9 -alsologtostderr
+FROM scratch  
```

Comment:
While I think it'd be great to lighten garg by using `scratch`, I'm not so sure it provides debuggability -- perhaps for now we use Alpine or some other "lightweight" base?

Reply:
Would it make sense to have a debug container built as well?  I really like scratch containers due to the small attack surface.  Even Alpine needs to be patched on a regular basis, and most certainly has zero days.  https://github.com/kubernetes/enhancements/issues/277 should also help with this when it hits by providing ephemeral containers.